src/cli/package/info.rs

```diff
@@ -1,13 +1,29 @@
 // SPDX-License-Identifier: Apache-2.0
 
+use crate::drawbridge::{client, TagSpec};
+
+use anyhow::Context;
 use clap::Args;
 
 /// Retrieve information about a published package.
 #[derive(Args, Debug)]
-pub struct Options {}
+pub struct Options {
+    #[clap(long)]
+    insecure_auth_token_file: Option<String>,
```

Comment:
So on `login` the token is stored in the keychain, but to actually use it, users have to manually retrieve it from the keychain and write it to a file?
I think this should be an environment variable and just a plaintext token there is fine IMO. (e.g. `ENARX_AUTH_TOKEN`) It would be very painful to specify this parameter for every `enarx` invocation

Reply:
Sure, although environment variables are quite a bit more insecure than even normal files are, and also now the insecure behavior is entirely implicit. Maybe we just deal with the pain for now and prioritize getting config files up?

Reply:
Environment variables are still the only thing e.g. a Github Action CI job would have.
Sure, it a job could write the value of the env var to a file and then use it by path everywhere, but that's terrible UX